docs/builders/iso.mdx

```diff
@@ -159,6 +159,10 @@ in the image's Cloud-Init settings for provisioning.
     not set, defaults to a random MAC. If value is "repeatable", value of MAC
     address is deterministic based on VM ID and NIC ID.
 
+  - `mtu` (int) - Set the maximum transmission unit for the adapter. Valid
+    range: 0 - 65520. If set to `1`, the MTU is inherited from the bridge
+    the adapter is attached to. Defaults to `0` (use Proxmox default).
```

Comment:
I couldn't find anything on that in the Proxmox documentation. To be honest I don't even know if Proxmox is setting the default, or if this is done by QEMU once the VM starts. In my tests I always ended up with an MTU of 1500, when was `mtu` unset.
But since that is also the the MTU of the underlying bridge, the default _might_ also be to just use the MTU of the connected bridge.